paws_collector.js

```diff
@@ -686,6 +686,13 @@ class PawsCollector extends AlAwsCollector {
     };
 
     _storeCollectionState(pawsState, privCollectorState, invocationTimeout, callback) {
+        // Added extra parameter to identify the message from which collector in case it failed and went into DLQ
+        let extraParam = {
+            lambda_function_name: process.env.AWS_LAMBDA_FUNCTION_NAME,
```

Comment:
Don't think we need function name either

Reply:
Okay